rustls/src/crypto/mod.rs

```diff
@@ -588,6 +566,46 @@ pub fn default_fips_provider() -> CryptoProvider {
     aws_lc_rs::default_provider()
 }
 
+mod crypto_default_provider {
```

Comment:
Maybe call this `default_impl`?

Reply:
I came up with `mod static_default`, can change it again if needed.